bin/powershell/check-windows-directory.ps1

```diff
@@ -0,0 +1,33 @@
+<#
+.SYNOPSIS 
+    Checks if  directory exist
+.DESCRIPTION
+    Checks if directory exist
+.Notes
+    FileName    : check-windows-directory.ps1
+    Author      : Patrice White - patrice.white@ge.com
+.LINK 
+    https://github.com/sensu-plugins/sensu-plugins-windows
+.PARAMETER LogName 
+    Required. The name of the directory.
+    Example -Dir C:\Users\dir
+.EXAMPLE
+    powershell.exe -file check-windows-directory.ps1 -Dir C:\Users\dir
+#>
+
+[CmdletBinding()]
+Param(
+  [Parameter(Mandatory=$True)]
+  [string]$Dir
+)
+
+$ThisDir = Test-Path -Path $Dir
+
+#Shows diretory if it exist
+if ($ThisDir) {
+    "CheckDirectory OK: Directory exist"
+    EXIT 0
+}else {
+    "CheckDirectory CRITICAL: Directory doesn't exist"
+    EXIT 2
+}
```

Comment:
I don't know if the convention holds true in windows setups but in `*nix` it is always best to include a newline at the end of your file. As I see your other 2 PRs have one I assume that this was just oversight rather than a convention in only `*nix` land.

Reply:
@majormoses Yes. It was just an oversight.